app/Http/Controllers/MismatchController.php

```diff
@@ -23,7 +23,7 @@ public function index(MismatchesRequest $request)
         // limit to 'pending',
         // unless include_reviewed parameter is provided
         if (!$request->boolean('include_reviewed')) {
-            $query->where('status', 'pending');
+            $query->where('review_decision', 'pending');
```

Comment:
was it decided to change it to `review_decision`. if my memory doesn't fail i remember we said `review_status` sounded better.

Reply:
Yes, I think you're right. Changed.